AsyncDisplayKit/ASMapNode.mm

```diff
@@ -161,7 +161,18 @@ - (MKCoordinateRegion)region
 
 - (void)setRegion:(MKCoordinateRegion)region
 {
-  self.options.region = region;
+  MKMapSnapshotOptions * __weak oldOptions = self.options;
+  MKMapSnapshotOptions * options = [[MKMapSnapshotOptions alloc] init];
+  options.camera = oldOptions.camera;
+  options.mapRect = oldOptions.mapRect;
+  options.mapType = oldOptions.mapType;
+  options.showsPointsOfInterest = oldOptions.showsPointsOfInterest;
+  options.showsBuildings = oldOptions.showsBuildings;
+  options.size = oldOptions.size;
+  options.scale = oldOptions.scale;
+  options.region = region;
+  self.options = options;
+//  self.options.region = region;
```

Comment:
@george-gw yes, @Adlai-Holler is right - we should copy the options, then just write to the region value, and set it back to self.options.